main-application/src/test/java/factory/LeftAileronFactoryTest.java

```diff
@@ -0,0 +1,15 @@
+package factory;
+
+import static org.junit.Assert.*;
+import org.junit.Test;
+
+
+public class LeftAileronFactoryTest {
+
+    @Test
+    public void testBuild(){
+        assertNotNull(LeftAileronFactory.build("1","test","3"));
```

Comment:
Leerzeichen nach den `,` ;)

Reply:
Bei IntelliJ bei dem committen kannst du rechts `code format`  (oder Ähnlich) setzen, dann macht der sowas automatisch bevor es gepusht wird. Oder in der Ordnerstruktur code format (oder so).